lib/schema-inspector.js

```diff
@@ -839,7 +839,14 @@
 		},
 		trim: function (post) {
 			return post.trim();
-		}
+		},
+		escapeXML: function (post) {
```

Comment:
let's use lowercase hyphen case (like-this insteadOfLikeThis) for rule names. It works better when you have multiple acronyms or initialisms. So `unescape-xml` etc.